[2023-12-07 22:20:33,366] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_redshift_docker_airflow.cargar_data manual__2023-12-07T22:15:17.479291+00:00 [queued]>
[2023-12-07 22:20:33,376] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_redshift_docker_airflow.cargar_data manual__2023-12-07T22:15:17.479291+00:00 [queued]>
[2023-12-07 22:20:33,376] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-07 22:20:33,377] {taskinstance.py:1377} INFO - Starting attempt 2 of 6
[2023-12-07 22:20:33,377] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-07 22:20:33,391] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): cargar_data> on 2023-12-07 22:15:17.479291+00:00
[2023-12-07 22:20:33,394] {standard_task_runner.py:52} INFO - Started process 1004 to run task
[2023-12-07 22:20:33,398] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_redshift_docker_***', 'cargar_data', 'manual__2023-12-07T22:15:17.479291+00:00', '--job-id', '188', '--raw', '--subdir', 'DAGS_FOLDER/ETL a Redshift.py', '--cfg-path', '/tmp/tmptt01uqrf', '--error-file', '/tmp/tmp253gqotf']
[2023-12-07 22:20:33,399] {standard_task_runner.py:80} INFO - Job 188: Subtask cargar_data
[2023-12-07 22:20:33,462] {task_command.py:371} INFO - Running <TaskInstance: ETL_redshift_docker_airflow.cargar_data manual__2023-12-07T22:15:17.479291+00:00 [running]> on host 4357a10b93bd
[2023-12-07 22:20:33,502] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-12-07 22:20:33,534] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ManuelMoyano
AIRFLOW_CTX_DAG_ID=ETL_redshift_docker_***
AIRFLOW_CTX_TASK_ID=cargar_data
AIRFLOW_CTX_EXECUTION_DATE=2023-12-07T22:15:17.479291+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-07T22:15:17.479291+00:00
[2023-12-07 22:20:33,535] {logging_mixin.py:115} INFO - Cargando la data para la fecha: 2023-12-07 22
[2023-12-07 22:20:35,899] {python.py:173} INFO - Done. Returned value was: None
[2023-12-07 22:20:35,911] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETL_redshift_docker_***, task_id=cargar_data, execution_date=20231207T221517, start_date=20231207T222033, end_date=20231207T222035
[2023-12-07 22:20:35,941] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-07 22:20:35,969] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
